src/main/target/STM32F3DISCOVERY/target.h

```diff
@@ -105,7 +105,7 @@
 #define MPU9250_CS_PIN          SPI2_NSS_PIN
 #define MPU9250_SPI_INSTANCE    SPI2
 // BMI160 gyro support
-//#define USE_ACCGYRO_BMI160
+#define USE_ACCGYRO_BMI160
```

Comment:
Is it intentional to enable this?

Reply:
Oops... I turned it on to make sure `USE_ACCGYRO_BMI160` compiles.

Reply:
But this is okay. I will keep it until gyro transition is over.